Makefile

```diff
@@ -25,22 +42,30 @@ fmt:
 
 .PHONY: build
 build:	
-	@bazel build //...
+	@bazel build ${BUILD_FLAGS} //...
+
+.PHONY: docker
+docker:
+	@bin/push-docker ${hub} ${tag}
 
 .PHONY: clean
 clean:
 	@bazel clean
 
 .PHONY: test
-test: build
-	@bazel test //...
+test:
+	@bazel test ${BUILD_FLAGS} //...
```

Comment:
I wonder: does this work fine on native mac OS? Should we restrict k8 to e2e test only?

Reply:
Yeah, its a bit of pain when this test starts e2e as well. if we restrict k8s to e2e alone, then it makes instructions and stuff easier. I could probably even code up the refresh goldens here.

Reply:
Actually, it works perfectly on Mac. Except the proxy/envoy test. And you don't even need the build flags. Updated PR